solr/test-framework/src/java/org/apache/solr/util/SolrJettyTestRule.java

```diff
@@ -31,18 +31,13 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+// TODO write javadocs
```

Comment:
you will find a ton of TODOs across Solr but no "nocommit".  "nocommit" is checked for in the build and the PR is unmergeable until these are addressed.